ksqldb-streams/src/main/java/io/confluent/ksql/execution/streams/materialization/ks/KsMaterialization.java

```diff
@@ -34,15 +35,18 @@
   private final Optional<WindowInfo> windowInfo;
   private final KsStateStore stateStore;
   private final Locator locator;
+  private final KsqlConfig ksqlConfig;
 
   KsMaterialization(
       final Optional<WindowInfo> windowInfo,
       final Locator locator,
-      final KsStateStore stateStore
+      final KsStateStore stateStore,
+      final KsqlConfig ksqlConfig
```

Comment:
I think you could skip adding the config as it is available in the stateStore already

Reply:
Yes, already did that. Thanks